src/Seq.App.EmailPlus/EmailApp.cs

```diff
@@ -39,19 +39,19 @@ internal EmailApp(IMailGateway mailGateway, IClock clock)
             _clock = clock ?? throw new ArgumentNullException(nameof(clock));
 
             // ReSharper disable ExpressionIsAlwaysNull ConditionIsAlwaysTrueOrFalse
-            _options = _options = new SmtpOptions
+            Options = new Lazy<SmtpOptions>(() => new SmtpOptions
             {
                 Server = Host,
+                DnsDelivery = DeliverUsingDns != null && (bool) DeliverUsingDns,
                 Port = Port ?? 25,
-                SocketOptions = EnableSsl ?? false
+                SocketOptions = EnableSsl != null && (bool) EnableSsl
                     ? SecureSocketOptions.SslOnConnect
-                    : SecureSocketOptions.StartTlsWhenAvailable,
-                Username = Username,
-                Password = Password,
+                    : SecureSocketOptions.None,
```

Comment:
Note - although the docs say that EnableSsl = false is equal to SecureSocketOptions.StartTlsWhenAvailable, the behaviour is actually SecureSocketOptions.None. As we don't have a dropdownlist that could offer the 3 options, I'm adding an extra bool to the app config to govern this behaviour.